source/_components/binary_sensor.urlwatch.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "URLWatch Binary Sensor"
```

Comment:
Unsure about  the name but if no feedback comes back on the parent PR it's ok as is.